rustls/src/crypto/mod.rs

```diff
@@ -186,6 +186,21 @@ pub struct CryptoProvider {
     pub key_provider: &'static dyn KeyProvider,
 }
 
+impl CryptoProvider {
+    /// Returns if this `CryptoProvider` is operating in FIPS mode.
+    pub fn fips_mode(&self) -> bool {
+        self.cipher_suites
```

Comment:
Should we do a `let Self { cipher_suites, .. }` thing here to make it more likely we'll catch field additions?

Reply:
Good idea!